armi/physics/executers.py

```diff
@@ -191,7 +191,7 @@ def run(self):
         if self.options.savePhysicsFiles:
             outputDir = os.path.join(pathTools.armiAbsPath(os.getcwd()), state, dirName)
         else:
-            outputDir = None
+            outputDir = pathTools.armiAbsPath(os.getcwd())
```

Comment:
**NOTE**: This line is added as an unrelated bug fix, to a line added [here](https://github.com/terrapower/armi/commit/d33b7f9978f857da538b37789082b6db48c00e9d#diff-4a5f481998dc37c62ef0f03e848e807414124dc5f9ffc742d9ece1f2ad52c5d4R194).